.azure-pipelines/stage/publish.yml

```diff
@@ -204,8 +231,101 @@ jobs:
           set -e
           rm -rf $(Build.StagingDirectory)/.gnupg
 
+- job: signed_bins
+  displayName: Signed binaries
+  dependsOn:
+  - package_x64
+  - package_arm64
+  condition: |
+    and(succeeded(),
+        eq(${{ parameters.runPackaging }}, 'true'))
+  timeoutInMinutes: 120
+  pool:
+    vmImage: "ubuntu-20.04"
+  steps:
+  - task: DownloadBuildArtifacts@0
+    inputs:
+      buildType: current
+      artifactName: "bazel.release"
+      itemPattern: "bazel.release/**/bin/*"
+      targetPath: $(Build.StagingDirectory)
+  - task: DownloadBuildArtifacts@0
+    inputs:
+      buildType: current
+      artifactName: "bazel.distribution"
+      itemPattern: "bazel.distribution/**/packages.*.tar.gz"
+      targetPath: $(Build.StagingDirectory)
+  - template: ../bazel.yml
+    parameters:
+      ciTarget: release.signed
+      stepsPre:
+      - template: ../gpg.yml
+        parameters:
+          authGPGPassphrase: ${{ parameters.authGPGPassphrase }}
+          authGPGPath: ${{ parameters.authGPGPath }}
+          authGPGKey: ${{ parameters.authGPGKey }}
+          pathGPGConfiguredHome: /build/.gnupg
+          pathGPGHome: $(Build.StagingDirectory)/.gnupg
+
+- job: docs
+  displayName: Publish docs
+  dependsOn: []
+  condition: |
+    and(not(canceled()),
+        eq(${{ parameters.publishDocs }}, 'true'))
+  pool:
+    vmImage: "ubuntu-20.04"
+  steps:
+  - template: ../bazel.yml
+    parameters:
+      ciTarget: docs
+      cacheTestResults: ${{ parameters.cacheTestResults }}
```

Comment:
there was a reason i added this altho struggling to rem what it was - i think we will want this but im not sure its immediately necessary